sdk/storage/Azure.Storage.Common/src/Shared/StorageVersionExtensions.cs

```diff
@@ -42,7 +42,7 @@ internal static class StorageVersionExtensions
         /// </summary>
         public const ServiceVersion LatestVersion =
 #if BlobSDK || QueueSDK || FileSDK || DataLakeSDK || ChangeFeedSDK || DataMovementSDK || BlobDataMovementSDK || ShareDataMovementSDK
-            ServiceVersion.V2024_02_04;
+            ServiceVersion.V2025_01_05;
```

Comment:
We probably need to revert this line, the recorded SAS tests will probably fail now.